### ISSUE TITLE
Questions about outer-loop loss.backward()

### ISSUE THREAD

**Issue by ghost** [Vision-CAIR/cs326-few-shot-classification#3]
I noticed that in the outer loop, the outer_loop_loss.backward() and optim.step() cannot update the MAMLModel.params, which causes each episode cannot update the model.params.

Finally, the meta part cannot change the model.params and has no effect.

I’m wondering whether this is deliberately designed? If yes, it seems that we need to add some code except the “TODO” to update the initial params using the gradients of fast weights?
